### DISCUSSION TITLE
Incorrect Training Intensity Times

### DISCUSSION THREAD

**Discussion by timede66** [philosowaffle/peloton-to-garmin#653] [TE / TSS / Vo2 / Intensity Minutes]
**Describe the bug**
It would appear that the training "intensity" minutes are not uploading correctly  

Comparing a normal outdoor bike ride of 60 minutes to a Peloton ride of 60 minutes 
The Garmin Edge 530 (with a heart monitor strap)  would record 45 "intensity" minutes for the outdoor ride
On a Peloton ride (with the same heart monitor strap) and uploading to Garmin it would show 10 "intensity" minutes

The rides would roughly correlate on Calories burnt, heart rates, cadence and times etc. 

**Expected behaviour**
I would expect some difference but not this big 

**Installation (please complete the following information):**
 - OS: windows 
 - Method: exe
 - Version: v4.1.0 runtime 7.0.14

**Comment by philosowaffle** [comment]
P2G does not calculate or upload Intensity Minutes.  This information is calculated by Garmin after it inspects the activity metrics (namely HR).  There appear to be[ a lot of different things](https://support.garmin.com/en-US/?faq=pNU9nnDzzGAHmEavp9rpY8) that go into Garmins "rules" for calculating Intensity Minutes.

Most interestingly, the device the activity was recorded on seems to change what requirements need to be met.  I would check:

1. Does your HR Zone information look correct on the uploaded P2G workout
2. Does your Power Zone information look correct on the uploaded P2G workout
3. Experiment configuring different Device types by changing [P2G's configuration](https://philosowaffle.github.io/peloton-to-garmin/latest/configuration/format/#customizing-the-garmin-device-associated-with-the-workout). Available on version [4.2.0](https://github.com/philosowaffle/peloton-to-garmin/releases/tag/v4.2.0). (more [context](https://github.com/philosowaffle/peloton-to-garmin/issues/606))

**Comment by timede66** [comment]
ok - thanks for the feedback
